### ISSUE TITLE
Arrival + withRipples

### ISSUE THREAD

**Issue by ghost** [mdbootstrap/mdb-ui-kit#367]
I think you forgot to add support for arrive.js and ripples elements,
I've added this

```
$(document).arrive(this.options.withRipples, function() {
 $.material.ripples($(this));
});
```

at the bottom of material.js and now it works

**Comment by siddhsuper** [comment]
Thanks for the immediate response.

I have taken a older version and its working for me.

**Comment by FezVrasta** [comment]
fixed with #365
